.env.sample

```diff
@@ -0,0 +1,11 @@
+# rename .env.sample to .env
+# .env files will be ignored by github
+
+# all values defined here can be used in react javascript code
+# like this without further imports: process.env.REACT_APP_BACKEND
+
+#################################
+# BACKEND DEVELOPMENT API SERVER
+# change to your personal raspiblitz IP & HTTP port (default is 80):
+#################################
+REACT_APP_BACKEND="http://192.168.1.100:80"
```

Comment:
Wanna add a new line here to satisfy the linting?

Reply:
fine with me

Reply:
Added it